### ISSUE TITLE
Paypal express integration

### ISSUE THREAD

**Issue by ilanbaron** [hyva-themes/magento2-hyva-checkout#45]
Integrate Paypal express checkout in the checkout following the Magento documentation. https://devdocs.magento.com/guides/v2.4/graphql/payment-methods/paypal-express-checkout.html

**Comment by schmengler** [comment]
WIP here: https://github.com/hyva-themes/magento2-hyva-checkout-paypal-express

**Comment by TheRealHunzik** [comment]
hello do we need another extension for Paypal integration?

i have installed hyve checkout module but paypal is not working

**Comment by progammer-rkt** [comment]
@TheRealHunzik Please see the documentation related to payment integration into hyvä checkout: https://hyva-themes.github.io/magento2-hyva-checkout/payment-integration/

Basically, payment methods are kind of "plugins" for hyvä checkout. Currently, these plugins are heavily under construction. We will be updating the documentation as we progress with more payment methods. Paypal-Express is something you can checkout at the time though. 

So in short, If you need any payment method with hyvä checkout, then at the moment, you need to integrate it your own.

**Comment by progammer-rkt** [comment]
@TheRealHunzik Please note the paypal-express payment repo is not available public at the moment. We will be publishing this soon. If you need an early access, then let me know

cc @wigman @schmengler

**Comment by TheRealHunzik** [comment]
Hello @progammer-rkt  Thanks for your time, actually we are working on a project using hyva theme if you can give us early access of PayPal integration i will be thankful to you :)

**Comment by progammer-rkt** [comment]
@TheRealHunzik This is not possible to give early access to the repo. It is right now in an immature state and sharing it leads to a lot of questions and bugs later on. I discussed this with the internal team and they suggest keeping it private until it matured.

Also, please keep in mind that, the Hyvä Checkout is a completely open-source project and it has no correlation with Hyvä Themes except for the fact that the team behind Hyvä Themes is working hard to make this full-featured, matured solution for Magento 2 checkout. 

I suggest you wait for a stable release of Hyvä Checkout. But, if you can take care of payment integrations and such challenges by yourself, then you can start to explore this solution :)

**Comment by DanielRuf** [comment]
Hi, what is the current status regarding thus? Any roadmap or ETA for Paypal?

**Comment by schmengler** [comment]
ping @mautz-et-tong

**Comment by mautz-et-tong** [comment]
ACK, see https://github.com/hyva-themes/magento2-hyva-checkout-paypal-express

**Comment by DanielRuf** [comment]
@mautz-et-tong it does not seem to be public

![image](https://user-images.githubusercontent.com/827205/144265340-6e9840b5-b2dd-4ea1-b5a0-8c80f3b7ae00.png)

**Comment by Muhamann** [comment]
Hello,

still the Paypal-Review-Page is NOT possible in Hyvä... this is the only page, our system has to do a Fallback to Luma.

Why Hyvä did not made this important, build-in payment method compatible completly, and still the review page is incompatible with Hyvä?

**Comment by Muhamann** [comment]
> > Hello,
> > still the Paypal-Review-Page is NOT possible in Hyvä... this is the only page, our system has to do a Fallback to Luma.
> > Why Hyvä did not made this important, build-in payment method compatible completly, and still the review page is incompatible with Hyvä?
> 
> Because the React Checkout is a community project and the PayPal compatibility was created by a contributor. Hyvä does not officially support/maintain this project.
> 
> The Hyvä Theme does not have a dedicated paypal-review-page because we found it unnecessarily complex (it's basically an entire checkout) and we implemented the paypal review differently in the Hyvä Checkout (the magewire one, not the React one), without the review page.


Yes, i understand this in general... but this Review Page is used by so many... and this is now the only page, i must implement the Blank or Luma Theme for... seems to make everything so "dirty" now.

What do you mean with: "_and we implemented the paypal review differently in the Hyvä Checkout (the magewire one, not the React one), without the review page_."
I use he Hyvä Checkout but i dont know how to use the Paypals Express-Buttons without the Review-Page? Can you explain, how i achieve that?

Thank you
